### PR TITLE
Add recipes for Mender Product 3.0.0 release.

### PR DESCRIPTION
**Pull request by kacf**
Changelog: Add mender-client-3.0.0 recipe.
Changelog: Add mender-artifact-3.6.0 recipe.
Changelog: Add mender-connect-1.2.0 recipe.
Changelog: Add mender-configure-1.0.1 recipe.

Signed-off-by: Kristian Amlie <kristian.amlie@northern.tech>

### CODE REVIEW

**Comment by kacf** [comment]
@mender-test-bot start pipeline

**Comment by kacf** [comment]
@mender-test-bot start pipeline

**Comment by kacf** [comment]
@mender-test-bot start pipeline

File: meta-mender-commercial/recipes-mender/mender-binary-delta/mender-binary-delta_1.1.1.bb [hunk 1]
```diff
@@ -1,5 +1,6 @@
 require mender-binary-delta.inc
 
-# Enable this in Betas, not in finals.
+# Enable this in Betas, and in branches that cannot carry this major version as
+# default.
```
Comment (lluiscampos) [inline meta-mender-commercial/recipes-mender/mender-binary-delta/mender-binary-delta_1.1.1.bb:4]: Nice rephrasing :+1:

**Pull request merged**